sdks/python/apache_beam/runners/direct/test_stream_impl.py

```diff
@@ -309,8 +310,8 @@ def is_alive():
       return not (shutdown_requested or evaluation_context.shutdown_requested)
 
     # The shared queue that allows the producer and consumer to communicate.
-    channel: Queue[Union[test_stream.Event,
-                         _EndOfStream]] = Queue()  # noqa: F821
+    channel: 'Queue[Union[test_stream.Event, _EndOfStream]]' = (  # noqa: F821
```

Comment:
does the entire hint need to be quoted or only '_EndOfStream' ?

Reply:
Also test_stream.Event. Given the noqa, it seems there were issues with this before.